vscode/microsoft-kiota/src/extension.ts

```diff
@@ -242,6 +243,77 @@ export async function activate(
         }
       }
     ),
+    registerCommandWithTelemetry(reporter,
+      `${treeViewId}.generateHttpSnippet`,
+      async () => {
```

Comment:
Since this is new code, and there's planned refactoring, it would be safe to move the functionality of this command out of the extension.ts file.

The command can follow the abstract command class as defined in this PR https://github.com/microsoft/kiota/pull/5160

Reply:
@thewahome I can wait for yours to get merged first to avoid increasing the conflict delta, if we both refactor at the same time, the conflict will be greater down the line

Reply:
It is likely not going to get merged because of how much the code has changed since freezing it. It's going to be maintained as a reference.